### PR TITLE
ADR 0010 ODH/Caikit/TGIS integration

### PR DESCRIPTION
**Pull request by Xaenalt**
Overview of the architecture and diagram of the ODH+Caikit+TGIS architecture

## Merge criteria:

- [X] The commits are squashed in a cohesive manner and have meaningful messages.
- [X] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [X] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by Xaenalt** [comment]
@jwforres @etirelli PTAL

File: ODH-ADR-0010-caikit-tgis-architecture.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Risks
+
+## Stakeholder Impacts
+
+
+| Group                             | Key Contacts                  | Date | Impacted? |
+| ----------------------------------- | ------------------------------- | ------ | ----------- |
+| RedHat Model Serving Team         | Sean Pryor, JooHo Lee         |      | Yes       |
+| RedHat Distributed Workloads Team | Anish Asthana                 |      | Yes       |
```
Comment (anishasthana) [inline ODH-ADR-0010-caikit-tgis-architecture.md:52]:
```suggestion
| RedHat Distributed Workloads Team | Anish Asthana, Antonin Stefanutti                 |      | Yes       |
```

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: ODH-ADR-0010-caikit-tgis-architecture.md (hunk 1 above)
Comment (anishasthana) [reply ODH-ADR-0010-caikit-tgis-architecture.md:52]: @astefanutti fyi

**Comment by Xaenalt** [comment]
> Imo we should have the lucidchart be included in this ADR itself

It's linked in "other docs" and in the "references" section, is there a way to include it more generally?

**Comment by astefanutti** [comment]
> > Imo we should have the lucidchart be included in this ADR itself
> 
> It's linked in "other docs" and in the "references" section, is there a way to include it more generally?

I'd recommend exporting the diagram to SVG, commit the SVG file along the ADR, and include the SVG into the ADR  markdown file as suggested by @anishasthana.

Also as Lucidchart is being retired, I'd recommend to export the diagram in VSDX format so it can be imported into other diagramming solutions like draw.io.

File: ODH-ADR-0010-caikit-tgis-architecture.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Open Data Hub - ODH, Caikit, and TGIS architecture
+
+
+|                |                                                                                                                              |
+| ---------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| Date           | 2023-Sept-13                                                                                                                 |
+| Scope          | OpenDataHub and Caikit/TGIS integration architecture                                                                         |
+| Status         | Accepted                                                                                                                     |
```
Comment (anishasthana) [inline ODH-ADR-0010-caikit-tgis-architecture.md:8]: Why is this saying accepted already? :-)

File: ODH-ADR-0010-caikit-tgis-architecture.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+| Authors        | [Sean Pryor](@Xaenalt),                                                                                                      |
+| Supersedes     | N/A                                                                                                                          |
+| Superseded by: | N/A                                                                                                                          |
+| Tickets        |                                                                                                                              |
+| Other docs:    | https://lucid.app/lucidchart/06fbfa85-ac66-40f7-9e60-1aa1d1ae426b/edit?invitationId=inv_74fb2b71-c771-405e-909c-e813b7d65623 |
+
+## What
+
+This ADR describes the architecture of the joint IBM-RedHat integration of ODH and Caikit/TGIS into the AI stack.
```
Comment (israel-hdez) [inline ODH-ADR-0010-caikit-tgis-architecture.md:17]: Are we good mentioning company ascription here?

It doesn't seem relevant for ODH.

**Comment by Xaenalt** [comment]
RHOAI + Caikit Architecture v1 0

**Pull request closed without merging**